graphql/schema.graphql

```diff
@@ -672,7 +672,8 @@ type TaskQueueItem {
 
 type TaskQueueDistro {
   id: ID!
-  queueCount: Int!
```

Comment:
we should keep this field to retain backwards compatability

Reply:
This is actually safe to remove it isn't used anywhere on the front end.

Reply:
queueCount is queried here: https://github.com/evergreen-ci/spruce/blob/main/src/pages/TaskQueue.tsx#L32-L35
as part of this page: https://spruce.mongodb.com/task-queue/rhel71-power8-small